server/src/modules/user_ensembles/dto/create-user-ensemble.dto.ts

```diff
@@ -2,9 +2,9 @@ import { IsNotEmpty } from 'class-validator';
 
 export class CreateUserEnseble {
```

Comment:
missing a b in ensemble

Reply:
I changed it but I actually think it is not even used anywhere